src/main/resources/application.yml

```diff
@@ -18,12 +19,11 @@ dynamodb:
 
 jobs:
   enabled: true
-  print-requests:
+  batch-print-requests:
     name: "BatchPrintRequests"
-    cron: "${JOB_PRINT_REQUESTS_CRON:0 0 */1 * * *}" # every hour by default
-#    lock-at-most-for: ${JOBS_PRINT_REQUEST_LOCK_AT_MOST_FOR}
-#    lock-at-least-for: ${JOBS_PRINT_REQUEST_LOCK_AT_LEAST_FOR}
-    batchSize: ${JOB_PRINT_REQUESTS_BATCH_SIZE}
+    cron: "0 0 */1 * * *" # every hour by default
+    lock-at-most-for: "PT10M"
+    batch-size: ${JOB_PRINT_REQUESTS_BATCH_SIZE}
```

Comment:
As with other properties here, do we think this should be an environment variable?
Sure, we will tune it and set an appropriate value; but do you think the value will be different between environments? 🤔

Reply:
We will need a default value if we want to get rid of this variable. @NathanRussellValtech

Reply:
Go with 100 for now